tools/geoCodes.js

```diff
@@ -12,7 +12,7 @@ const options = {
 
 const geocoder = NodeGeocoder(options);
 
-const locations = Array.from(new Set(events.map((event) => event.location.replace(" & Online", "")))).filter((location) => !cachedGeolocations[location]).filter((location) => location !== "").sort();
+const locations = Array.from(new Set(events.map((event) => event.location.replace(" & Online", "")))).filter((location) => !cachedGeolocations[location]).filter((location) => location !== "" && location !== "Online").sort();
```

Comment:
You may want to do `&& location.toLowerCase() !== "online"` to account for casing

Reply:
Actually, my expectation is that every line in the Readme has the same format, so I expect that to be case sensitive, anyway, changing it to case insensitive is a bit more reslient to accidental introduction of "online" string without uppercase in the readme.

Reply:
Done, thanks for the suggestion. :slightly_smiling_face: